warehouse/query-core/src/main/java/datawave/query/jexl/JexlASTHelper.java

```diff
@@ -1867,6 +1868,14 @@ public static boolean validateJunctionChildren(JexlNode node, boolean failHard)
         return valid;
     }
 
+    public static NodeTypeCount getIvarators(JexlNode node) {
+        return NodeTypeCountVisitor.countNodes(node, QueryPropertyMarker.MarkerType.EXCEEDED_VALUE, QueryPropertyMarker.MarkerType.EXCEEDED_OR);
```

Comment:
See QueryPropertyMarker.IVARATOR_TYPES, in addition to these two types the EXCEEDED_TERM is also an ivarator type.